docs/project/development.md

```diff
@@ -260,6 +260,37 @@ $ git checkout <hash>
 
 ## Troubleshooting
 
+### 'span' file not found
+
+On Linux systems, the Clang compiler typically uses the `libstdc++` C++ standard library by default. `libstdc++` is the default C++ Standard Library implementation provided by the GNU Compiler Collection (GCC). While Clang may link against the `libc++` library, this requires explicitly providing the `-stdlib` flag when running Clang.
+
+Bun relies on C++20 features like `std::span`, which are not available in GCC versions lower than 11. GCC 10 doesn't have all of the C++20 features implemented. As a result, running `make setup` may fail with the following error:
+
+```bash
```

Comment:
can you remove `bash` from here, since it's not bash script.

(the markdown renderer will format this weirdly)